Architecture changes (#13)
ietf-scitt/draft-birkholz-scitt-architecture: draft-birkholz-scitt-architecture.md

Feed:

> : An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Registryon a Transparency Service contains a sequence of Claims about the same Artifact.

Reviewer: missing space
```suggestion
: An identifier chosen by the Issuer for the Artifact. For every Issuer and Feed, the Registry on a Transparency Service contains a sequence of Claims about the same Artifact.
```

 In COSE, Feed is one header attributes in the protected header of the Envelope.

Registry:

: the verifiable data structure that stores Claims in a transparency service. SCITT supports multiple Registry formats to accommodate different transparency service implementations, such as historical Merkle Trees and sparse Merkle Trees.

Transparency Service:

: the entity that maintains and extends the Registry, and endorses its state. A Transparency Service can be a complex distributed system, and SCITT requires the TS to provide many security guarantees about its Registry . The identity of a TS is captured by a public key that must be known by Verifiers in order to validate Receipts.

Receipt:

: a Receipt is a special form of COSE countersignature for Claims that embeds cryptographic evidence that the Claim is recorded in the Registry . It consists of a Registry -specific inclusion proof, a signature by the Transparency Service of the state of the Registry , and additional metadata (contained in the countersignature protected headers) to assist in auditing.

Registration:

> : the process of submitting a Claim to a Transparency Service, applying its registration policy, storing it in the Registryand producing the Receipt returned to the submitter.

Reviewer: missing space here, too
```suggestion
: the process of submitting a Claim to a Transparency Service, applying its registration policy, storing it in the Registry and producing the Receipt returned to the submitter.
```

                      v        signature  |     |
                    Claim  <--------------/     |
                      |                         |
                      |  Claim      +---------+ |
                      |------------>| Trans.  | |

> Transparency ->       +<------------| Regisrty| /

Reviewer: typo: Regisrty -> Registry
```suggestion
Transparency ->       +<------------| Registry| /
```

For example,
- a small, simple SCITT instance may keep track specifically of the software used for operating SCITT services.
- an air-gapped data center may operate its own SCITT Registry to retain full control and auditing of its software supplies.

How?

> - Policy-based. Within an organization, local Verifiers contact an authoritative SCITT that records the latest policies associated with classes of Artifacts; these policies indicate which Issuers and Registeries are trusted for verifying signed Transparent Claims for these Artifacts.

Reviewer: typo: Registeries -> Registries
```suggestion
- Policy-based. Within an organization, local Verifiers contact an authoritative SCITT that records the latest policies associated with classes of Artifacts; these policies indicate which Issuers and Registries are trusted for verifying signed Transparent Claims for these Artifacts.
```
Registries

- Other federation mechanisms?

> We'd like to attach multiple Receipts to the same signed Claims, each Receipt endorsing the Issuer signature and a subset of prior Receipts. This involves down-stream Registeries verifying and recording these Receipts before issuing their own Receipts.

Reviewer: typo: Registeries -> Registries
```suggestion
We'd like to attach multiple Receipts to the same signed Claims, each Receipt endorsing the Issuer signature and a subset of prior Receipts. This involves down-stream Registries verifying and recording these Receipts before issuing their own Receipts.
```
